From b04a125c80472f268d947aa6b2dc56dae1e4054c Mon Sep 17 00:00:00 2001
From: Zorro Liu <lyx@rock-chips.com>
Date: Wed, 20 Jan 2021 15:01:10 +0800
Subject: [PATCH] Revert "arm64: configs: rockchip_defconfig: add rockchip ebc
 dev driver"

This reverts commit 18a4299e0b9db716d6432db58baeed2c440ea3b4.

move to rk356x_eink.config on android which one will overlay kernel configs

Signed-off-by: Zorro Liu <lyx@rock-chips.com>
Change-Id: If43d748cdef541915a218214c274375a2d8e34bd
---
 arch/arm64/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index b5bb1b752a44..289120d7de14 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -590,7 +590,6 @@ CONFIG_ROCKCHIP_INNO_HDMI=y
 CONFIG_ROCKCHIP_LVDS=y
 CONFIG_ROCKCHIP_DRM_TVE=y
 CONFIG_ROCKCHIP_RGB=y
-CONFIG_ROCKCHIP_EBC_DEV=y
 CONFIG_DRM_ROCKCHIP_RK618=y
 CONFIG_DRM_ROCKCHIP_RK628=y
 CONFIG_DRM_PANEL_SIMPLE=y
-- 
2.35.3

